homeassistant/components/gryfsmart/entity.py

```diff
@@ -0,0 +1,73 @@
+"""Define the base entity for the Gryf Smart integration."""
+
+from __future__ import annotations
+
+from pygryfsmart.api import GryfApi
+from pygryfsmart.device import _GryfDevice
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity import Entity
+
+from .const import CONF_DEVICE_DATA
+
+
+class _GryfSmartEntityBase(Entity):
+    """Base Entity for Gryf Smart."""
+
+    _attr_should_poll = False
+    _attr_entity_registry_enabled_default = True
+
+    _api: GryfApi
+    _device: _GryfDevice
+    _attr_unique_id: str | None
+
+    @property
+    def name(self) -> str:
+        return self._device.name
+
+
+class GryfConfigFlowEntity(_GryfSmartEntityBase):
```

Comment:
config flows are to create config entries. There is no such thing as a config flow entity.